charts/stable/shinobi/values.yaml

```
@@ -0,0 +1,62 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: shinobisystems/shinobi
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: dev
```

```suggestion
  tag:
```
Please leave this empty as default. That will allow us to only update the Chart.yaml file if we ever need to update it.